cryptoconditions/condition.py

```diff
@@ -184,43 +249,60 @@ def max_fulfillment_length(self):
         Return:
              (int) Maximum length (in bytes) of any fulfillment payload that fulfills this condition..
         """
-        if not isinstance(self._max_fulfillment_length, int):
+        if not isinstance(self._cost, int):
             raise ValueError
-        return self._max_fulfillment_length
+        return self._cost
 
-    @max_fulfillment_length.setter
-    def max_fulfillment_length(self, value):
+    # TODO update docstrings
+    @cost.setter
+    def cost(self, value):
```

Comment:
If we want to be consistent we should to `cost` the same we did to `hash`. Check that the value has the correct type on the `setter` and and raise a `ValueError` on the `property` if the value is not set.